src/main/java/org/zendesk/client/v2/model/Request.java

```diff
@@ -26,6 +29,10 @@
     protected Date createdAt;
     protected Date updatedAt;
     protected Comment comment;
+	protected boolean solved;
```

Comment:
Please maintain project code formatting.

Reply:
Better to use object boolean here or solved will always default to false and may inadvertently change the entity state.